### DISCUSSION TITLE
WIKI: Brush Names - in the eaxmple the Background Brush is assigned to the Foreground property

### DISCUSSION THREAD

**Discussion by DrCQ** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3537] [Q&A]
In the Brush Names description in Wiki I found the following example:
`<TextBlock Foreground="{DynamicResource MaterialDesign.Brush.Primary}" />`

If I really understand the description above, it should be better to assign foreground brush to foreground property:
`<TextBlock Foreground="{DynamicResource MaterialDesign.Brush.Primary.Foreground}" />`

Do you agree?

**Comment by Keboo** [comment]
This can sort of depend on usage a little bit. The "Primary"/"Secondary" colors are the actual "colorful" theme colors in your application. Often for things like headings it is common to leverage those theme colors. The "*.Foreground" colors are a dark/light (gray scale; functionally white/black) contrasting color that will look good as a foreground on top of the theme color. For longer sections of text, they are what you would want to use. So, I think which you pick depends on how much text we are talking about, and where it is used in the application.
